app/views/service_providers/show.html.erb

```diff
@@ -13,14 +13,14 @@
   <label for="name">App name: </label>
   <span class="font-mono-xs margin-left-5"><%= service_provider.app_name %></span>
 </h2>
-<p><i> <%= I18n.t('service_provider_form.app_name') %> </i></p>
+<p><i> <%= sanitize I18n.t('service_provider_form.app_name') %> </i></p>
```

Comment:
Another approach is when we have HTML inside of i18n'd strings is to suffix the key with `_html` so that I18n will auto-HTML safe it

More docs on it: https://guides.rubyonrails.org/i18n.html#using-safe-html-translations

Reply:
The bonus is that the templates don't need to worry about this (aside from changing the key name)